app/assets/v2/js/pages/bounty_detail/casper_extension.js

```diff
@@ -0,0 +1,87 @@
+const payWithCasperExtension = async(fulfillment_id, to_address, vm, modal) => {
+
+  const amount = vm.fulfillment_context.amount;
+  const token_name = vm.bounty.token_name;
+  const tenant = vm.getTenant(token_name, vm.bounty.web3_type);
+
+  const { CasperClient, DeployUtil, PublicKey, Signer } = casper;
+  const casperClient = new CasperClient(`/api/v1/bounty/payout/broadcast/${tenant}`);
+
+  const isConnected = await Signer.isConnected();
+
+  if (!isConnected) {
+    try {
+      Signer.sendConnectionRequest();
+    } catch (e) {
+      _alert({ message: gettext('Please download or enable CasperLabs Signer extension.') }, 'danger');
+    }
+  }
+
+  const selectedAddress = await Signer.getActivePublicKey();
+
+  const paymentAmount = 10000;
```

Comment:
What's this for ? worth adding documentation ?

Reply:
done